### PR TITLE
Add recipe for CDC with Debezium.

### PR DESCRIPTION
**Pull request by morsapaes**

### CODE REVIEW

File: aggregations-and-analytics/09/09_cdc_materialized_view.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# 09 Maintaining Materialized Views with Change Data Capture (CDC) and Debezium
+
+:bulb: This example will show how you can use Flink SQL and Debezium to maintain a materialized view based on database changelog streams.
+
+In the world of analytics, databases are still mostly seen as static sources of data — like a collection of business state just sitting there, waiting to be queried. The reality is that most of the data stored in these databases is continuously produced and continuously changing, so...why not _stream_ it?
```
Comment (sjwiesman) [inline aggregations-and-analytics/09/09_cdc_materialized_view.md:7]:
```suggestion
In the world of analytics, databases are still mostly seen as static sources of data — like a collection of business states just sitting there, [waiting for queries](https://en.wikipedia.org/wiki/Waiting_for_Godot). In reality, while the tables may only show a snapshot of the state of the world at a particular point in time, rows are being continuously created and modified, so ... so not _stream_ it?
```

File: aggregations-and-analytics/09/09_cdc_materialized_view.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# 09 Maintaining Materialized Views with Change Data Capture (CDC) and Debezium
+
+:bulb: This example will show how you can use Flink SQL and Debezium to maintain a materialized view based on database changelog streams.
+
+In the world of analytics, databases are still mostly seen as static sources of data — like a collection of business state just sitting there, waiting to be queried. The reality is that most of the data stored in these databases is continuously produced and continuously changing, so...why not _stream_ it? 
+
+Change Data Capture (CDC) allows you to do just that: track and propagate changes in a database based on its changelog (e.g. the WAL in Postgres) to downstream consumers. [Debezium](https://debezium.io/) is a popular tool for CDC, and is supported in Flink through **1)** the [Kafka SQL Connector](https://ci.apache.org/projects/flink/flink-docs-stable/dev/table/connectors/formats/debezium.html) or **2)** a set of "standalone" [Flink CDC Connectors](https://github.com/ververica/flink-cdc-connectors#flink-cdc-connectors).
```
Comment (sjwiesman) [inline aggregations-and-analytics/09/09_cdc_materialized_view.md]:
```suggestion
Change Data Capture (CDC) allows you to do just that: track and propagate changes in a database based on its changelog (e.g., the [write-ahead log](https://www.postgresql.org/docs/current/wal-intro.html) in Postgres) to downstream consumers. [Debezium](https://debezium.io/) is a popular tool for CDC that Flink supports through **1)** the [Kafka SQL Connector](https://ci.apache.org/projects/flink/flink-docs-stable/dev/table/connectors/formats/debezium.html) or **2)** a set of "standalone" [Flink CDC Connectors](https://github.com/ververica/flink-cdc-connectors#flink-cdc-connectors)
```

File: aggregations-and-analytics/09/09_cdc_materialized_view.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+       SUM(claim_total) AS agg_claim_costs
+FROM accident_claims
+WHERE claim_status <> 'DENIED'
+GROUP BY accident_detail;
+```
+
+How can you check that the CDC functionality is _actually_ working? The `docker` directory also includes a data generator script (`postgres_datagen.sql`) with a series of `INSERT` operations with new insurance claims; if you run it, you can see how the query results update in (near) real-time:
+
+`cat ./postgres_datagen.sql | docker exec -i flink-cdc-postgres psql -U postgres -d postgres`
+
```
Comment (sjwiesman) [inline aggregations-and-analytics/09/09_cdc_materialized_view.md:72]: I feel like this is missing one wrap-up sentence. I need to think about it. There should be a conclusion and call to action.

File: aggregations-and-analytics/09/09_cdc_materialized_view.md (hunk 2 above)
Comment (morsapaes) [reply aggregations-and-analytics/09/09_cdc_materialized_view.md]: C'mon, Seth, let Debezium shine on its own.

File: aggregations-and-analytics/09/09_cdc_materialized_view.md (hunk 3 above)
Comment (morsapaes) [reply aggregations-and-analytics/09/09_cdc_materialized_view.md:72]: Agreed. Will add it in!

**Comment by morsapaes** [comment]
Thanks for the review (as always), Seth! Merging and putting it out in the world.

**Pull request merged**